deploy/docs/additional_prometheus_configuration.md

```diff
@@ -271,15 +268,13 @@ To send custom metrics to Sumo Logic you need to update it to include a rule to
 Make sure you include the same tag you created in your Fluentd configmap in the previous step.
 Here is an example addition to the configuration file that will forward metrics to Sumo:
 
+NOTE: Please add the below to a user-supplied values file  (```helm install -f```)
```

Comment:
incomplete command

Reply:
ok makes sense, I could keep the one comment at the beginning and remove the rest of them. Would that work?